src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/PreviousMinerBalanceFiller.yml

```diff
@@ -0,0 +1,152 @@
+      nonce: 1
+      code: :raw 0x60FF00
+      storage: {}
+
+  # Contract with 0 gas code execution.
+    0xBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB:
+      balance: 0x10000000000
+      nonce: 1
+      code: ''
+      storage: {}
+
+    0xd02d72E067e77158444ef2020Ff2d325f929B363:
+      balance: 0x10000000000000000
+      nonce: 1
+      code: ''
+      storage: {}
+
+    #Store miner tip at block finalization.
+    0xcccccccccccccccccccccccccccccccccccccccc:
+      balance: 0x10000000000
+      nonce: 1
+      code: |
+         :yul { sstore(sub(number(),1), sub(balance(coinbase()), sload(0xFFFF)) ) //Store the current miner tip (current_miner_balance - previous_miner_balance).
+                sstore(0xFFFF, balance(coinbase()) )                              //Record current miner balance for next block to compare.
+         }
+      storage: #Set all test balance storage slots to starting value 1.
+          0x01: 1
+          0x02: 1
+          0x03: 1
+          0x04: 1
+
+  blocks:
+  # Block 1, no miner tip.
+  - transactions:
+    - data: ''
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+
+  # Block 2, tip = 1 (gas limit 21000) with PUSH opcode (3 gas), (21000*1) + 3 = 21003.
+  - transactions:
+    - data: ''
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+    # This transaction gives the miner 21003 wei because it uses 21003 gas
+    # 21k for the transaction, and 3 for the push that AA..AA does
+    - data: ''
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0x01
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+      value: 0x0
+
+  # Block 3, tip = 1 (gas limit 21000) with no opcode (0 gas), (21000*1) + 0 = 21000.
+  - transactions:
+    - data: ''
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
+      value: 0x0
+    # This transaction gives the miner 21000 wei because it uses 21000 gas with no opcode.
+    - data: ''
+      accessList: []
+      gasLimit: 100000
+      maxPriorityFeePerGas: 0x01
+      maxFeePerGas: 1000
+      nonce: auto
+      secretKey: 41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298
+      to: 0xBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB
+      value: 0x0
```

Comment:
Can you add here one more transaction that checks balance of coinbase?
To compare it with the first transaction it must be identical.